integration_tests/src/main/python/conditionals_test.py

```diff
@@ -138,9 +138,7 @@ def test_ifnull(data_gen):
                 'ifnull({}, b)'.format(null_lit),
```

Comment:
2021 copyrights

Reply:
@firestarman would you mind trying the automated copyright update https://github.com/NVIDIA/spark-rapids/blob/branch-0.6/CONTRIBUTING.md#pre-commit-hooks

Reply:
Thanks a lot @gerashegalov. Great tool, will try it.

Reply:
nice catch. Updated